core/src/main/scala/io/chrisdavenport/http4sotel4s/package.scala

```diff
@@ -0,0 +1,17 @@
+package io.chrisdavenport
+
+import org.http4s.{Header, Headers}
+import org.typelevel.ci.CIString
+import org.typelevel.otel4s.{TextMapGetter, TextMapUpdater}
+
+package object http4sotel4s {
+  implicit val headersTMU: TextMapUpdater[Headers] =
+    (carrier: Headers, key: String, value: String) => carrier.put(Header.Raw(CIString(key), value))
+  implicit val headersTMG: TextMapGetter[Headers] =
+    new TextMapGetter[Headers] {
+      def get(carrier: Headers, key: String): Option[String] =
+        carrier.get(CIString(key)).map(_.head.value)
+      def keys(carrier: Headers): Iterable[String] =
+        carrier.headers.view.map(_.name).distinct.map(_.toString).toSeq
+    }
```

Comment:
these should ideally be in http4s itself, but that may not happen for quite a bit as otel4s isn't nearly as stable as http4s yet. should be perhaps have a couple modules, like `http4s-otel4s-core` and `http4s-otel4s-middleware`, and these could live in a core module until they make it into http4s proper?